### PR TITLE
Config alignment

### PR DESCRIPTION
**Pull request by DariusIMP**
Adding the following functionalities to Config.kt:

- `config.getJson(key: String): Result<String>`
- `config.insertJson5(key: String, value: String): Result<Unit>`
- `config.id(): ZenohID`
- `Config.fromEnv(): Result<Config>` (attempts to load the config on the path provided under the environment variable `ZENOH_CONFIG`)

### CODE REVIEW

File: zenoh-jni/src/config.rs [hunk 1]
```diff
@@ -119,6 +120,81 @@ pub extern "C" fn Java_io_zenoh_jni_JNIConfig_00024Companion_loadYamlConfigViaJN
     })
 }
 
+/// Obtains the id of the config, returning it as a little endian byte array.
+#[no_mangle]
+#[allow(non_snake_case)]
+pub unsafe extern "C" fn Java_io_zenoh_jni_JNIConfig_00024Companion_getIdViaJNI(
```
Comment (fuzzypixelz) [inline zenoh-jni/src/config.rs:126]: `Config::id` is not part of the API, it's only available with `unstable`.

**Review by fuzzypixelz — COMMENTED** [review]
`fromJson` and `fromYaml` aren't present in the Rust API, so I'm not sure if they should be kept.

File: zenoh-jni/src/config.rs (hunk 1 above)
Comment (DariusIMP) [reply zenoh-jni/src/config.rs]: Ok, removed it. For some confusion I thought it was available

**Comment by DariusIMP** [comment]
> `fromJson` and `fromYaml` aren't present in the Rust API, so I'm not sure if they should be kept.

These are kotlin specific functions that were the result of a discussion we had.

**Pull request merged**